spec/features/user_sign_in_spec.rb

```diff
@@ -6,18 +6,17 @@
     scenario 'I can sign in if I am a user in another instance' do
       other_instance = create(:instance)
       password = 'abcdefghi'
-      user = create(:user, password: password)
       ActsAsTenant.with_tenant(other_instance) do
-        other_instance.instance_users.create(user: user)
+        @user = create(:user, password: password)
```

Comment:
right, because there's no Instance User created before that.

Can you use user = nil in the old line 9 and not use the instance variable?